metron-deployment/vagrant/full-dev-platform/Vagrantfile

```diff
@@ -53,7 +53,7 @@ hosts = [{
 Vagrant.configure(2) do |config|
 
   # all hosts built on centos 6
-  config.vm.box = "bento/centos-6.7"
+  config.vm.box = "metron/centos_base"
```

Comment:
How did our Atlas environment change?  It seems like we now create our one CentOS image?  

Seems like a good idea being that Bento's image could and did change without warning.  Do I build the CentOS image with the rest of the `packer-build` stuff?